TodoApi/Program.cs

```diff
@@ -36,6 +36,19 @@
     return Results.Created($"/todo/{todo.Id}", todo);
 });
 
+app.MapPut("/todos/{id}", async ([FromServices] TodoDbContext db, int id, Todo todo) =>
+{
+    var todo = await db.Todos.FindAsync(id);
+    if (todo is null)
+    {
+        return Results.NotFound();
+    }
+    await db.Update(todo);
```

Comment:
TIL

Reply:
Huh, that was my oversight. Should be without `await` :| Only `AddAsync` can be awaited. I'll send a follow-up PR with a fix.

Reply:
Ah, I noticed that you fixed it, thanks.

Reply:
I already fixed it 😄